appcues/src/main/java/com/appcues/ui/composables/CompositionLocals.kt

```diff
@@ -94,11 +95,22 @@ internal sealed class StackScope(private val childrenCount: Int) {
     class ColumnStackScope(val width: Double?, childrenCount: Int) : StackScope(childrenCount)
 }
 
+/**
+ * Use LocalAppcuesStepMetadata to access the shared information between steps
+ *
+ * Example usage:
+ * @sample com.appcues.trait.appcues.BackdropKeyholeTrait
+ */
 val LocalAppcuesStepMetadata = compositionLocalOf<AppcuesStepMetadata> { noLocalProvidedFor("LocalAppcuesStepMetadata") }
 
+/**
+ * AppcuesStepMetadata used to share trait information between traits, generated from produceMetadata.
+ * previous and current refers to the the previous generated metadata and current as sometimes you may want to
+ * consult both values to enable for nice transition between the two.
```

Comment:
Suggestion: 
> Previous and current refer to the previously generated metadata values and the current values. It may be desired to reference both values to support transitions between them.